### ISSUE TITLE
exception on non-matching `textshader` 

### ISSUE THREAD

**Issue by qTich** [renpy/renpy#6026]
```renpy
  File "renpy/display/render.pyx", line 170, in renpy.display.render.render
    cpdef render(d, object widtho, object heighto, double st, double at):
  File "renpy/display/render.pyx", line 260, in renpy.display.render.render
    rv = d.render(widtho, heighto, st, at)
  File "/home/tichq/Downloads/renpy-sdkarm/renpy/text/text.py", line 2612, in render
    layout = Layout(self, width, height, renders, splits_from=virtual_layout)
  File "/home/tichq/Downloads/renpy-sdkarm/renpy/text/text.py", line 1020, in __init__
    mr = self.create_mesh_displayable(o, tex, lines, xo, yo, depth, max_depth, ts)
  File "/home/tichq/Downloads/renpy-sdkarm/renpy/text/text.py", line 1844, in create_mesh_displayable
    if (g.shader is ts) or (g.shader == ts):
  File "/home/tichq/Downloads/renpy-sdkarm/renpy/text/shader.py", line 176, in __eq__
    return self.key == other.key
AttributeError: 'NoneType' object has no attribute 'key'
```

For example in `tutorial` add any `textshader` for `style hyperlink_text`: 
```renpy
style hyperlink_text:
    properties gui.text_properties("hyperlink", accent=True)
    hover_underline True
    textshader "wave"
```
and open `About`.

**Comment by renpytom** [comment]
Please post full tracebacks, not elided versions.

**Comment by renpytom** [comment]
Also, is config.default_textshader set? This isn't a great message, but I believe that it can only trigger when config.default_textshader is None, and a default textshader is required to use any other textshaders.

**Comment by qTich** [comment]
Right, `config.default_textshader is None`.

I think it would be better to put the mixing information in a red box as done for `Live2D Web/iOS`.

Or check in advance and explicitly tell the user about it instead of the current exception.

**Comment by renpytom** [comment]
I agree, this was just a case that wasn't covered.

**Comment by renpytom** [comment]
Resolved by 41458e274ed163159d035cbd3c8526f669f2d3c5 .
